LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -48,7 +48,7 @@ public class LoRaDevice : IDisposable, ILoRaDeviceRequestQueue
 
         public AppKey? AppKey { get; set; }
 
-        public string AppEUI { get; set; }
+        public JoinEui? AppEUI { get; set; }
```

Comment:
Should we use this as an opportunity to rename `AppEUI` to the more conventional `AppEui`?

Reply:
Done in 6339ab6c5009e7292142037060b8e8c60128f308